frappe/database/__init__.py

```diff
@@ -11,40 +12,52 @@
 def setup_database(force, verbose=None, mariadb_user_host_login_scope=None):
 	import frappe
 
-	if frappe.conf.db_type == "postgres":
-		import frappe.database.postgres.setup_db
-
-		return frappe.database.postgres.setup_db.setup_database()
-	else:
+	if frappe.conf.db_type == "mariadb":
 		import frappe.database.mariadb.setup_db
 
 		return frappe.database.mariadb.setup_db.setup_database(force, verbose, mariadb_user_host_login_scope)
+	elif frappe.conf.db_type == "sqlite":
+		import frappe.database.sqlite.setup_db
+
+		return frappe.database.sqlite.setup_db.setup_database(force, verbose)
+	else:
+		import frappe.database.postgres.setup_db
+
+		return frappe.database.postgres.setup_db.setup_database()
```

Comment:
Just a quick comment, **_possibly_** a bug here in an edge-case. @akhilnarang 

What happens if `db_type` is not specified in the site config? Before, it would do nothing because most users use MariaDB and the `else:` case did fallback to MariaDB. But now, the `else` case fallthrough is PostgreSQL.

Reply:
Actually, probably not even from there, [this](https://github.com/frappe/frappe/blob/develop/frappe/commands/site.py#L24) gets propagated through to `make_site_config`

Reply:
Oh nice! Thanks for the info :smiley: